assignment/week_3/RecommendedMenu/src/components/SelectCategory.jsx

```diff
@@ -0,0 +1,59 @@
+import styled from "styled-components";
+
+const SelectCategory = (props) => {
+  const clickCategoryHandler = (e) => {
+    props.setSelectedCategory(e.target.innerHTML);
+  };
+
+  return (
+    <St.CategoryContainer>
+      <St.Category
+        onClick={(e) => {
+          clickCategoryHandler(e);
+        }}
+      >
+        취향대로 추천
+      </St.Category>
+      <St.Category
+        onClick={(e) => {
+          clickCategoryHandler(e);
+        }}
+      >
```

Comment:
```suggestion
      <St.Category
        onClick={clickCategoryHandler}
      >
        취향대로 추천
      </St.Category>
      <St.Category
        onClick={clickCategoryHandler}
      >
```

인수로 이벤트 객체만을 넘겨주는 경우엔 이렇게만 써도 알아서 넘겨준답니당 !